asterius/src/Asterius/Builtins/CMath.hs

```diff
@@ -0,0 +1,247 @@
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE HexFloatLiterals #-}
+{-# LANGUAGE OverloadedLists #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Asterius.Builtins.CMath
+  ( cmathCBits,
+  )
+where
+
+import Asterius.Types
+
+cmathCBits :: AsteriusModule
+cmathCBits = scalbn
+
+scalbn :: AsteriusModule
```

Comment:
Could you add a comment here indicating the provenance of this code (as in the MR description) and why we shouldn't try to understand it?